dask_glm/algorithms.py

```diff
@@ -97,22 +98,21 @@ def gradient_descent(X, y, max_iter=100, tol=1e-14, family=Logistic):
         Xbeta = Xbeta - stepSize * Xgradient
 
         if stepSize == 0:
-            print('No more progress')
             break
 
         df = lf - func
         df /= max(func, lf)
 
         if df < tol:
-            print('Converged')
             break
         stepSize *= stepGrowth
         backtrackMult = nextBacktrackMult
 
     return beta
 
 
-def newton(X, y, max_iter=50, tol=1e-8, family=Logistic):
+@normalize()
+def newton(X, y, max_iter=50, tol=1e-8, family=Logistic, **kwargs):
     '''Newtons Method for Logistic Regression.'''
```

Comment:
I started working on docs last night, I'll have something by the end of the week / early next week.